sips/sip-temporary_keypairs-des-128.md

```diff
@@ -0,0 +1,123 @@
+| SIP-Number          | <Leave this blank; it will be assigned by a SIP Editor>                                        |
+| ---:                |:-----------------------------------------------------------------------------------------------|
+| Title               | Keypairs in files should be aes-128 encode                                                     |
+| Description         | Currently Sui store keypair in file only using base64 which should using aes-128 encode better |
+| Author              | oday0311@hotmail.com                                                                           |
+| Editor              | <Leave this blank; it will be assigned by a SIP Editor>                                        |
+| Type                | Standard                                                                                       |
+| Category            | Core                                                                                           |
+| Created             | 2024-05-03                                                                                     |
+| Comments-URI        | <Leave this blank; it will be assigned by a SIP Editor>                                        |
+| Status              | <Leave this blank; it will be assigned by a SIP Editor>                                        |
+| Requires            | <Optional; SIP number(s), comma separated>                                                     |
+
+## Abstract
+
+Sui currently store keypairs in files[sui.keystore,network.yaml,fullnode.yaml] only using base64 which should using aes-128 encode better.
+
+## Motivation
+
+base64 is not secure enough to store keypair in files, it should use aes-128 encode better.
+there are some cases, we don't want to display the base64 directly in the files,
+by add the default aes-encode, developers can compile the bin files with their own encrypt password, 
+so instead of save base64 content , I think aes-encode content with user-custom aes-128 result is more reasonable. 
+in fact, similar cases exist in the config files, network.yaml, fullnode.yaml, all the key-pair currently only save as base-64,
+but key-pairs content is less used in normal developer, but cli-key tool is widly use for 
+dapps developers in their server for deploy dapps.
+
+## Specification
```

Comment:
this section should describe what the AES encoding for keystore do, the code should go to the reference implementation section, or simply a link to your pr). 

i.e. describe what's the prefix used for encoding, how is the password set, how does AES encryption work (which standard does it follow with link). example of what a key looks like before and after encryption.